### PR TITLE
[docs] Update Sui CLI install instructions to include the tracing feature

### PR DESCRIPTION
**Pull request by tzakian**
## Description 

Update docs for installing the Sui CLI to include `--features tracing` so that they work with the trace viewer/debugger.

## Test plan 

👀

### CODE REVIEW

**Review by tedks — APPROVED** [review]
LGTM!

**Review by awelc — APPROVED** [review]
LGTM, but shouldn't we include some explanation on what the feature flag does (or a link to an explanation)?

File: docs/content/guides/developer/getting-started/sui-install.mdx [hunk 1]
```diff
@@ -112,7 +112,7 @@ To confirm that Sui installed correctly, type `sui --version` in your console or
 Run the following command to install Sui binaries from the `testnet` branch:
 
 ```shell
-cargo install --locked --git https://github.com/MystenLabs/sui.git --branch testnet sui
+cargo install --locked --git https://github.com/MystenLabs/sui.git --branch testnet sui --features tracing
```
Comment (stefan-mysten) [inline docs/content/guides/developer/getting-started/sui-install.mdx:115]: Let's add an explanation what tracing is and when / when not to use it. It does not hurt to be more doc complete here!

File: docs/content/guides/developer/getting-started/sui-install.mdx [hunk 2]
```diff
@@ -130,7 +130,7 @@ The command installs Sui components in the `~/.cargo/bin` folder.
 If you previously installed the Sui binaries, you can update them to the most recent release with the same command you used to install them (changing `testnet` to the desired branch):
 
 ```shell
-cargo install --locked --git https://github.com/MystenLabs/sui.git --branch testnet sui
+cargo install --locked --git https://github.com/MystenLabs/sui.git --branch testnet sui --features tracing
```
Comment (stefan-mysten) [inline docs/content/guides/developer/getting-started/sui-install.mdx:133]: Ditto!

**Pull request merged**